owtf/managers/resource.py

```diff
@@ -105,7 +105,7 @@ def get_resources_from_file(resource_file):
         resource_file, "r"
     ).read().splitlines()  # To remove stupid '\n' at the end
     for line in config_file:
-        if "#" == line[0]:
+        if line.startswith("#"):
```

Comment:
`line` here is a list not a string

Reply:
Actually `line` is a string inside `config_file` which is a list, and so, `startwith` function is running fine without error.
![str](https://github.com/owtf/owtf/assets/54772821/e31bdbc2-07d0-4a25-b2ca-cda0d1d9d55e)

Reply:
Actually `line` is a string inside `config_file` which is a list, and so, `startwith` function is running fine without error.
![str](https://github.com/owtf/owtf/assets/54772821/e31bdbc2-07d0-4a25-b2ca-cda0d1d9d55e)